### PR TITLE
bedFormat scale nameFloat

### PR DESCRIPTION
**Pull request by rimangan**
Added testing for bedFormat, which previously lacked testing coverage. Implemented a new option: scaleNameFloat. In some cases, we have stored float values in the name field. This option simply allows the user to scale all the values stored in the name column by a constant multiplier.

### CODE REVIEW

File: cmd/bedFormat/bedFormat.go [hunk 1]
```diff
@@ -40,11 +49,19 @@ func usage() {
...
+	EnsemblToUCSC  bool
+	ScaleNameFloat float64
+}
+
 func main() {
 	var expectedNumArgs int = 2
 	var ensemblToUCSC *bool = flag.Bool("ensemblToUCSC", false, "Changes chromosome format type.")
 	var UCSCToEnsembl *bool = flag.Bool("UCSCToEnsembl", false, "Changes chromosome format type.")
-	var fields *int = flag.Int("fields", 4, "Specify the number of bed fields.")
+	var scaleNameFloat *float64 = flag.Float64("scaleNameFloat", math.MaxFloat64, "If float values are held in the name field, scale those values by this constant multiplier.")
```
Comment (craiglowe) [inline cmd/bedFormat/bedFormat.go:64]: Should this scale value default to 1?

**Review by craiglowe — APPROVED** [review]
Great to have this.  Is 1 a better default option for scaling all numbers?  This seems like it will be more understandable to the user who sees it as the default value in the help message.

File: cmd/bedFormat/bedFormat.go (hunk 1 above)
Comment (rimangan) [reply cmd/bedFormat/bedFormat.go:64]: We need an option to not scaleNameFloat for beds where non-float64 data is stored in the name field

**Review by cfauci — APPROVED** [review]

**Review by RavenYL — APPROVED** [review]

**Review by craiglowe — APPROVED** [review]
Great work!

**Pull request merged**